src/backend/Modules/Account/Timebox.Account.Application/Services/EmailService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Net.Mail;
+using Timebox.Account.Application.Interfaces.Services;
+
+namespace Timebox.Account.Application.Services
+{
+    public class EmailService : IEmailService
+    {
+        public bool IsValidEmailAddress(string email)
+        {
+            if (string.IsNullOrWhiteSpace(email))
+                return false;
+
+            try
+            {
+                // ReSharper disable once CA1806
+                new MailAddress(email);
```

Comment:
Can also use the MailAddressAttribute class to do this

Reply:
On the parameter to this method? Do you think this should be changed?

Reply:
No no the library will be fine, just a could have done if you get me don't need to change anything

Reply:
Okay! Cheers